internal/pkg/service/buffer/watcher/apinode/apinode.go

```diff
@@ -74,7 +75,7 @@ func New(d Dependencies, opts ...Option) (*Node, error) {
 		clock:  d.Clock(),
 		logger: d.Logger().AddPrefix("[api][watcher]"),
 		client: d.EtcdClient(),
-		stats:  statistics.NewCollectorNode(d),
+		stats:  d.StatsCollector(),
```

Comment:
A tu bol dalsi :lady_beetle: , namiesto nacitania instancie z `dependencies` sa tu vytvarala nova.
- Nemalo to vplyv na funckiu, ale blbo sa debugovalo, preco su vsetky logs 2x.